SchedulerBot/SchedulerBot.Database.Core/UnitOfWork.cs

```diff
@@ -11,57 +9,57 @@ namespace SchedulerBot.Database.Core
 	/// <inheritdoc/>
 	public class UnitOfWork : IUnitOfWork
 	{
-		private readonly DbContext dbContext;
+		private readonly SchedulerBotContext dbContext;
 		private readonly ILifetimeScope lifetimeScope;
 
-		public UnitOfWork(ILifetimeScope lifetimeScope, DbContext dbContext)
+		/// <summary>
+		/// Initializes a new instance of the <see cref="UnitOfWork"/> class.
+		/// </summary>
+		/// <param name="dbContext">The database context.</param>
+		/// <param name="lifetimeScope">The lifetime scope.</param>
+		public UnitOfWork(SchedulerBotContext dbContext, ILifetimeScope lifetimeScope)
 		{
 			this.dbContext = dbContext;
 			this.lifetimeScope = lifetimeScope;
+
+			ManageConversationLinks = GetRepository<IManageConversationLinkRepository>();
```

Comment:
It would be nice to use lazy loading for repositories.

Reply:
Done